simapp/simd/cmd/root.go

```diff
@@ -81,6 +84,19 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 	return rootCmd, encodingConfig
 }
 
+// initTendermintConfig helps to override default Tendermint Config values.
+// return tmcfg.DefaultConfig if no custom configuration is required for the application.
+func initTendermintConfig() *tmcfg.Config {
+	cfg := tmcfg.DefaultConfig()
+
+	cfg.P2P.MaxNumInboundPeers = 100
+	cfg.P2P.MaxNumOutboundPeers = 40
+	cfg.Mempool.Size = 10000
+	cfg.StateSync.TrustPeriod = 112 * time.Hour
```

Comment:
Are these values better defaults for app devs in general? Maybe we should add some comments on why we chose them over `tmcfg.DefaultConfig()`?

Reply:
commented them out but left a comment.